nvflare/private/fed/server/fed_server.py

```diff
@@ -191,9 +191,20 @@ def deploy(self, args, grpc_args=None, secure_train=False):
         cleanup_thread.start()
 
     def client_cleanup(self):
+        last_remove_time = 0.0
+        remove_interval = 5.0
+        check_interval = 0.2
```

Comment:
This is good improvement.

As we keep saying, later we need to have a common place for all these random constants sleep / check time interval.

Reply:
Without this, client_cleanup can be stuck for 15 seconds, which will cause the MPM's final cleanup to fail.